doc/query.md

```diff
@@ -0,0 +1,545 @@
+# Cloudant Query - Android
+
+Cloudant Query - Android is a Java implementation of [Cloudant Query][1] for Android mobile devices. It is included as a sub-package of [sync-android][2].
+
+Cloudant Query is inspired by MongoDB's query implementation, so users of MongoDB should feel at home using Cloudant Query in their mobile applications.
+
+The aim is that the query you use on our cloud-based database works for your mobile application.
+
+**Note**: this is a very early version, and it probably won't work as you expect yet! In
+particular, there's very little information to help debugging when errors happen, leaving you
+in the debugger. There is also little error checking, so things might explode without warning.
```

Comment:
Removed in https://github.com/cloudant/sync-android/commit/d5b4f7825035187c3ef55b5962f38b66806f68d3.  Will make a note to remove from objc version.